packages/asc-ui/src/components/Checkbox/CheckboxStyle.ts

```diff
@@ -1,6 +1,5 @@
 import styled, { css } from 'styled-components'
```

Comment:
There is still some styling present that modifies the `outline` in this file.

Reply:
We need the outline rules here as well.

Reply:
:+1: